lib/order.rb

```diff
@@ -0,0 +1,32 @@
+require_relative 'dish'
+require_relative 'menu'
+
+class Order
+  attr_reader :order_list, :dish
+
+  def initialize(dish = Dish, menu = Menu)
```

Comment:
I'm slightly confused about how you would use this in irb if you were specifiying a dish/menu